### ISSUE TITLE
Backup Station does not work

### ISSUE THREAD

**Issue by q-pa** [ghostbsd/issues#97]
### Version

24.01.1

### What desktop?

Official MATE

### Explain the issue and what happens

Trying to start Backup Station from GUI -> nothing seems to happen.
Start from CLI ->
```
Traceback (most recent call last):
  File "/usr/local/bin/backup-station", line 210, in confirm_passwd
    BackupStationWindow().show_all()
  File "/usr/local/bin/backup-station", line 114, in __init__
    self.create_be_list()
  File "/usr/local/bin/backup-station", line 25, in create_be_list
    self.liststore.append(element.split())
  File "/usr/local/lib/python3.9/site-packages/gi/overrides/Gtk.py", line 1016, in append
    return self._do_insert(-1, row)
  File "/usr/local/lib/python3.9/site-packages/gi/overrides/Gtk.py", line 1007, in _do_insert
    row, columns = self._convert_row(row)
  File "/usr/local/lib/python3.9/site-packages/gi/overrides/Gtk.py", line 899, in _convert_row
    raise ValueError('row sequence has the incorrect number of elements')
ValueError: row sequence has the incorrect number of elements
```

### Explain how to reproduce the bug?

do the above

**Comment by ericbsd** [comment]
I moved the ticket to get fixed for 24.04.1, but it should have been fixed before that release.

**Comment by ericbsd** [comment]
The update for the fix will be available tomorrow.
